packages/hooks/src/useMT5SVGEligibleToMigrate.ts

```diff
@@ -0,0 +1,65 @@
+import { useMT5AccountsList } from '@deriv/api';
+import { Jurisdiction, getFormattedJurisdictionCode } from '@deriv/shared';
+
+const useMT5SVGEligibleToMigrate = () => {
+    const { data: mt5_login_list = [] } = useMT5AccountsList();
+    const svg_accounts_to_migrate = mt5_login_list.filter(
+        account => account.landing_company_short === 'svg' && !!account.eligible_to_migrate
+    );
+
+    const has_svg_accounts_to_migrate = !!svg_accounts_to_migrate.length;
+    const no_of_svg_accounts_to_migrate = svg_accounts_to_migrate.length;
+
+    const is_eligible_for_svg_to_bvi_migration = !!svg_accounts_to_migrate.filter(account =>
+        Object.values(account.eligible_to_migrate || {}).includes(Jurisdiction.BVI)
+    ).length;
+
+    const is_eligible_for_svg_to_vanuatu_migration = !!svg_accounts_to_migrate.filter(account =>
+        Object.values(account.eligible_to_migrate || {}).includes(Jurisdiction.VANUATU)
+    ).length;
+
+    const getEligibleAccountToMigrate = () => {
+        let account_to_migrate = '';
+        if (is_eligible_for_svg_to_bvi_migration) {
+            account_to_migrate = Jurisdiction.BVI;
+        } else if (is_eligible_for_svg_to_vanuatu_migration) {
+            account_to_migrate = Jurisdiction.VANUATU;
+        }
+        return account_to_migrate;
```

Comment:
just a doubt, based on BE response, there will never be case where both `is_eligible_for_svg_to_bvi_migration` and `is_eligible_for_svg_to_vanuatu_migration` will be true, right?